index.html

```diff
@@ -233,7 +250,7 @@ <h1 class="header-title text-center">Contact me</h1>
                    placeholder="Enter your name here" 
                    value="Abdallah" required>
             <input class="input txt-input" type="email" name="email" placeholder="Enter your email here" value="abdallahantony55.aa@gmail.com" required>
```

Comment:
- [ ] Kindly remove the value attribute from the input elements. They are not necessary for this case.